llvm/lib/CodeGen/AsmPrinter/DwarfDebug.h

```diff
@@ -625,6 +629,12 @@ class DwarfDebug : public DebugHandlerBase {
   /// Emit the reference to the section.
   void emitSectionReference(const DwarfCompileUnit &CU);
 
+  /// Populate dependent type variable map
+  void populateDependentTypeMap();
```

Comment:
Didn't you comment out the definition of this function in `llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp`?

Reply:
Fixed.